src/bash/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+!.gitignore
```

Comment:
This line is repeated in many places? Can you remove?

Reply:
It's technically not necessary since the .gitignore file is itself added. I added it in all the files because I assume in the future, if someone adds a new directory they will use one of the existing .gitignore files as a template. However, If the file doesn't un-ignore itself, it will not show up in their list of modified files, so it will not get added to their commit.

I can simply remove them all if you prefer.